cert-generation/main.go

```diff
@@ -120,8 +120,7 @@ func main() {
 		}
 	}
 
-	// Wait until it's cleaned up by the workspace-controller
-	log.Println("Certs have been successfully created. Waiting to be cleaned up by devworkspace controller")
+	log.Println("Certs have been successfully created.")
```

Comment:
Just idea for future: if possible - that deployment could listen to created service removing and recreate it after it's removed. Otherwise - it needs to be restarted to restore broker installation. But it's not critical for that PR and maybe even does not worth to be implemented.